src/Runner/Linter.php

```diff
@@ -82,13 +89,18 @@ private function fix(iterable $finder, Ruleset $ruleset): void
 
         foreach ($finder as $file) {
             $filePath = $file->getPathname();
-            $success = $fixer->fixFile($filePath);
+            $contents = file_get_contents($filePath);
+            if (false === $contents || !$this->cacheManager->needFixing($filePath, $contents)) {
```

Comment:
if `false === $contents` I think an Exception should be thrown.

Reply:
Done deal